bindings/c#/examples/Rotating3DCube/Program.cs

```diff
@@ -7,7 +7,17 @@
 const float LerpPow = 0.002f;
 const int ChangePerFrames = 50;
 
-using var leds = new RGBLedMatrix(32, 1, 1);
+// Set this to apropriate settings for your matrix
+var options = new RGBLedMatrixOptions()
+{
+    Brightness= 80,
+    Cols= 64,
+    Rows= 64,
+    GpioSlowdown= 4
+};
+
+using var leds = new RGBLedMatrix(options);
```

Comment:
Everywhere else it's `matrix`, not `leds`. When unifying the initialization in the samples, it would make sense to refactor that variable naming here too ;)